accounts/views.py

```diff
@@ -350,7 +350,7 @@ def edit_email_settings(request):
     else:
         # Get list of enabled email_types
         all_emails = request.user.profile.get_enabled_email_types()
```

Comment:
I did not add it because this was implicitly done in the tests that trigger emails and for which we check whether these are sent or not (the tests I added in this PR). However I'll also add a test for `get_enabled_email_types` method.

Reply:
Added f96a9c451ebc69e761baa633a33472e974a014d5